apps/web/components/ui/Toolbar/Toolbar.vue

```diff
@@ -1,37 +1,43 @@
 <template>
   <div class="sticky top-0 bg-white py-2 z-50">
-    <div class="flex justify-center md:justify-end pr-5 space-x-2">
-      <UiButton
-        variant="secondary"
-        :size="buttonSize"
-        class="self-start"
-        @click="toggleEdit"
-        :disabled="!isEditingEnabled"
-        data-testid="edit-preview-button"
-      >
-        {{ disableActions ? 'Preview' : 'Edit' }}
-      </UiButton>
-      <UiButton
-        variant="primary"
-        :size="buttonSize"
-        class="self-start"
-        :disabled="!isEditingEnabled || !isLocalTemplate()"
-        data-testid="edit-save-button"
-        @click="updatePageTemplate"
-      >
-        <template v-if="loading">
-          <SfLoaderCircular class="animate-spin w-4 h-4" />
-        </template>
+    <div class="relative flex items-center pr-5">
+      <div class="absolute left-1/2 transform -translate-x-1/2" v-if="localeCodes.length > 1">
+        <UiLanguageEditor />
```

Comment:
**todo**: Please add an experimental flag. I know this is a small feature, but we should practice this to get used to it.